src/modules/map/components/cesium/CesiumPopover.vue

```diff
@@ -24,55 +29,58 @@ export default {
     },
     watch: {
         coordinates() {
+            this.updateCoordinateHeight()
             this.updatePosition()
         },
     },
     mounted() {
-        this.updatePosition()
         const viewer = this.getViewer()
         if (viewer) {
+            this.cesiumHandler = new ScreenSpaceEventHandler(viewer.scene.canvas)
+            this.cesiumHandler.setInputAction(this.updatePosition, ScreenSpaceEventType.MOUSE_MOVE)
+            // wheel event will trigger a zoom, the tooltip will need repositioning
+            this.cesiumHandler.setInputAction(this.updatePosition, ScreenSpaceEventType.WHEEL)
+            // rotating the view/camera is done by CTRL+click and move, so we also listen to mouse move when CTRL is down
+            this.cesiumHandler.setInputAction(
+                this.updatePosition,
+                ScreenSpaceEventType.MOUSE_MOVE,
+                KeyboardEventModifier.CTRL
+            )
```

Comment:
this solution works really well, I didn't think that problem can be because of the height. As the problem is only with the height we can remove these handlers and revert `camera.percentageChanged` (tested, works well, zooming even smoother) if there are no special cases I didn't notice